test/tools/htmlreport/check.sh

```diff
@@ -26,21 +26,22 @@ if [ -z "$PYTHON" ]; then
     PYTHON=python
 fi
 
+PROJECT_ROOT_DIR=$(git rev-parse --show-toplevel)
 REPORT_DIR=$(mktemp -d -t htmlreport-XXXXXXXXXX)
 INDEX_HTML="$REPORT_DIR/index.html"
 STATS_HTML="$REPORT_DIR/stats.html"
 GUI_TEST_XML="$REPORT_DIR/gui_test.xml"
 ERRORLIST_XML="$REPORT_DIR/errorlist.xml"
 UNMATCHEDSUPPR_XML="$REPORT_DIR/unmatchedSuppr.xml"
 
-$PYTHON cppcheck-htmlreport --file ../gui/test/data/xmlfiles/xmlreport_v2.xml --title "xml2 test" --report-dir "$REPORT_DIR" --source-dir ../test/
```

Comment:
Ew. Glad we moved that.

Reply:
Should this be also removed from the script?